browser/src/Services/Learning/Tutorial/Tutorials/TargetsVimPluginTutorial.tsx

```diff
@@ -42,25 +42,33 @@ export class TargetsVimPluginTutorial implements ITutorial {
             new Stages.SetBufferStage([Line1a, Line1b, Line1c]),
             new Stages.SetCursorPositionStage(0, 0),
             new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 2, 46),
-            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.MoveToGoalStage("Type 'foo'", 2, 49),
+            new Stages.WaitForModeStage("Press ESC to go back to normal mode", "normal"),
             Stages.combine(
                 null,
                 new Stages.FadeInLineStage(null, 1, Line2a),
                 new Stages.FadeInLineStage(null, 2, Line2b),
-                new Stages.SetBufferStage([Line2a, Line2b, Line2c]),
+                new Stages.FadeInLineStage(null, 3, Line2c),
+                new Stages.SetBufferStage([Line2a, Line2b, Line2c, Line2d]),
             ),
             new Stages.SetCursorPositionStage(0, 7),
             new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 0, 40),
```

Comment:
Minor, but I just realized `paranthesis` was misspelled a couple of places - should be `parenthesis`.

Reply:
Good catch.